src/create_character.cpp

```diff
@@ -147,7 +181,7 @@ void PickBgState::update()
 
     case MenuAction::esc:
     {
-      pop_states(4);
+      pop_until(states::StateId::MENU_STATE);
```

Comment:
Hm, this looks like it shouldn't work?

`pop_until` is in the `states` namespace - shouldn't it be `states::pop_until()` ?